contribute/creating_package.md

```diff
@@ -55,30 +52,32 @@ The package called "tiddlywinks" can now be created using the template `t` by ru
 t("tiddlywinks")
 ```
 
-As the folder has not been specified, the package has been created at `~/.julia/dev` on Linux, and at `username/julia/dev` in Windows. If this is not the repository you want to be working from, move the files to the original repo which you have created. Now in `tiddlywinks.jl` you should have
+Last line in the REPL shows where the new package has been created:
+`Info: New package is at *package_repo*`. As the folder has not been specified, the package has been created at `~/.julia/dev` on Linux, and at `username/julia/dev` on Windows. If this is not the repository you want to be working from, move the files to the original repo which you have created. Now in `tiddlywinks.jl` you should have
```

Comment:
```suggestion
`Info: New package is at *package_repo*`. As the folder has not been specified, the package has been created at `~/.julia/dev` on Linux, and at `%USERNAME%/.julia/dev` on Windows. If this is not the repository you want to be working from, move the files to the original repo which you have created. Now in `tiddlywinks.jl` you should have
```

It is still `.julia` using `%USERNAME%` allows one to directly copy paste it into e.g. Explorer.

Reply:
done, thanks